### PR TITLE
Update passwords info

### PR DESCRIPTION
**Pull request by sarawilcox**
## Description
- Update passwords info on page about publishing prototypes
- Change 2 mentions of NHSD to NHS England
- 
## Checklist

- [ ] Tested against our [testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [x] CHANGELOG entry

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 # NHS.UK prototype kit Changelog
 
-## 4.8.7 - TBC February 2023
+## 4.8.7 - TBC May 2023
```
Comment (the-nathan-smith) [inline CHANGELOG.md]: Do we still need to "TBC" here or can this be removed?

Comment (sarawilcox) [reply CHANGELOG.md]: Hi @the-nathan-smith. We'll put in the exact date when we know what it is. E.g. 29 May 2023.

**Review by pflynny — APPROVED** [review]
LGTM

**Review by roshaanbajwa — APPROVED** [review]
looks good to me !

**Review by roshaanbajwa — APPROVED** [review]

**Pull request merged**